packages/waas/src/auth.ts

```diff
@@ -37,7 +37,15 @@ import {
   MaySentTransactionResponse,
   SignedMessageResponse
 } from './intents/responses'
-import { WaasAuthenticator, AnswerIncorrectError, Chain, EmailAlreadyInUseError, Session } from './clients/authenticator.gen'
+import {
+  WaasAuthenticator,
+  AnswerIncorrectError,
+  Chain,
+  EmailAlreadyInUseError,
+  Session,
+  WebrpcEndpointError,
+  WebrpcError
+} from './clients/authenticator.gen'
```

Comment:
Is this change necessary? What uses `WebrpcEndpointError` and `WebrpcError` in this file?

Reply:
yep you are right, I think I forgot to remove after testing